pages/dashboard.tsx

```diff
@@ -159,16 +156,20 @@ export default function DashboardPage() {
                     <Td width="5%">{org.id}</Td>
                     <Td>
                       {org.name}
-                      <Tag
-                        className="whitespace-nowrap flex-shrink-0 justify-center ml-2"
-                        size={"md"}
-                        key={index}
-                        borderRadius="full"
-                        variant="solid"
-                        colorScheme={getColorLabel(org?.subName)}
-                      >
-                        {translateSubName(org?.subName)}
-                      </Tag>
+                      {parseBooleanEnvValue(
+                        window.__env.NEXT_PUBLIC_STRIPE_ENABLED
+                      ) && (
```

Comment:
Why do you access the `window.__env` directly instead of using `loadConfig` which uses it to configure the configuration values?